pkg/services/ngalert/api/tooling/definitions/alertmanager_validation.go

```diff
@@ -101,6 +101,21 @@ func (r *Route) Validate() error {
 	return r.validateChild()
 }
 
+func (r *Route) ValidateMuteTimes(muteTimes map[string]struct{}) error {
```

Comment:
What happens in this scenario:
1. User A opens the UI and seeds mute timings for a route, with timings B and C.
2. User B deletes mute timing B with an API call.
3. User A removes mute timing C from the Route and calls the API.

The error is "mute time interval B does not exist", right? What's their next step? Reload and remove timing C again?

Reply:
Step 2. would already fail, as you can not delete a mute timing that is still in use. The missing piece is the other way round, you reference a mute timing that was never created.